config/v1/types_apiserver.go

```diff
@@ -30,6 +30,18 @@ type APIServerSpec struct {
 	// - ConfigMap.Data["ca-bundle.crt"] - CA bundle.
 	// +optional
 	ClientCA ConfigMapNameReference `json:"clientCA"`
+	// extraCORSAllowedOrigins lists additional, user-defined regular expressions describing hosts for which the
+	// API server allows access using the CORS headers. This is needed to access the API and the integrated OAuth
+	// server from JavaScript applications.
+	// The field values should follow the syntax of the corsAllowedOrigins field. This means that the values are
```

Comment:
I would drop everything below about operators. This is not the doc for the operator writes but for users.

Reply:
Removed, added the one sentence about operator changes of the field to the design.

Reply:
Which `corsAllowedOrigins` field is meant here? Do we need that reference? Isn't the sentence about the regex syntax enough?

Reply:
I'll keep just the regex syntax sentence